src/main/kotlin/racingcar/Car.kt

```diff
@@ -1,16 +1,27 @@
 package racingcar
 
 class Car(
-    private var location: Int = 0,
+    private var _location: Int = DEFAULT_CAR_LOCATION,
+    val name: String,
 ) {
-    fun move(randNumber: Int) {
-        val rule = MovingJudgeRule.judge(randNumber)
-        val strategy = rule.strategy()
+    init {
+        require(isValidLength(name)) { "글자수는 5자를 넘을 수 없어요" }
```

Comment:
객체가 생성되는 시점에 유효성을 체크하도록 만드셨군요? 👍👍
개인적으로 지금의 경우에는 Validation 로직이 간단하므로 메서드로 추출할 필요없이 바로 명시하는 것이 더 가독성이 좋은 것 같아요. 🙂 

추가적으로 예외 메시지를 조금 더 명확하게 적어보면 어떨까요? 😃

```suggestion
        require(name.length <= VALID_CAR_NAME_LENGTH) { "자동차 이름은 5자를 넘을 수 없어요" }
```